src/tutorial/runtime/reinstalling.rst

```diff
@@ -0,0 +1,155 @@
+.. _tutorial.Reinstalling-workflows:
+
+Fixing and Reloading Workflows
+==============================
+
+.. admonition:: Aim
+
+   Use ``cylc reinstall`` and ``cylc reload`` to repair a broken workflow
+   configuration without having to run the whole workflow again.
```

Comment:
This sentence could be construed as implying that those are the only two options. Probably worth noting that shutdown, reinstall, restart is an alternative to reload too.

Reply:
```suggestion

... note::

   This tutorial explores one working practice. You might, alternatively, wish
   to use ``cylc shutdown``, ``cylc reinstall`` and ``cylc play`` to modify and restart
   an existing workflow.

```